src/components/item/ItemMain.tsx

```diff
@@ -13,6 +15,11 @@ import ItemMetadataContent from './ItemMetadataContent';
 import ItemPanel from './ItemPanel';
 import ItemHeader from './header/ItemHeader';
 
+const ItemContainer = emotionStyled.div<{ isMainMenuOpen: boolean }>`
+  width: ${({ isMainMenuOpen }) =>
+    isMainMenuOpen ? 'calc(100vw - 240px)' : '100vw'};
+`;
+
```

Comment:
I think it's better to follow the following syntax:
```
const StyledContainer = styled(Box)<{ open: boolean }>
```
With `'div'` instead of `Box`.